### PR TITLE
Added pitch compensation for columbus view

### PR DESCRIPTION
**Pull request by robdan7**
Fixes the use case described in #9183 

`CesiumMath.lerp(startPitch, pitch, time)` has been replaced with the same `pitchFunction(time)` which is already used in 3D view.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @robdan7!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/master/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by robdan7** [comment]
CLA has been submitted!

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @robdan7! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Review by ebogo1 — COMMENTED** [review]
Thanks @robdan7 - I confirmed this fixes the flight path in CV mode and the change looks good to me. I had a minor question about the double entry in CONTRIBUTORS.md, but otherwise this should be ready to merge.

If it's easier I'd be happy to resolve the merge conflict and add an entry to CHANGES.md to get this PR over the finish line.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -278,3 +278,5 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 - [Gusain Vipul](https://github.com/vipulgusain)
 - [Sam Bakkach](https://github.com/sambakk)
 - [Zoran Kokeza](https://github.com/zoran995)
+- [Robin Danielsson](https://github.com/robdan7)
```
[the remark this answers is not in the document]
Comment (robdan7) [reply CONTRIBUTORS.md:281]: Yes, this was a cooperative effort. we were unaware of co-authors at the time.

**Comment by ebogo1** [comment]
@robdan7 I pushed a couple commits to get this ready for merge - should be good to go once CI passes.

**Comment by ebogo1** [comment]
Thanks @robdan7!

**Pull request merged**